### PR TITLE
Changed name from 'SNIC' to 'NAISS' in acknowledgements

### PR DESCRIPTION
**Pull request by sofiahag**

### CODE REVIEW

File: data/report_templates/project_summary.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ such as publications, presentations, posters, etc., using the following statemen
 
 > The authors acknowledge support from the National Genomics Infrastructure in {% if project.ngi_facility -%} {{ project.ngi_facility }} {%- endif %}
 > funded by Science for Life Laboratory, the Knut and Alice Wallenberg Foundation and the Swedish Research Council,
-> and SNIC/Uppsala Multidisciplinary Center for Advanced Computational Science for assistance with massively parallel sequencing
+> and NAISS/Uppsala Multidisciplinary Center for Advanced Computational Science for assistance with massively parallel sequencing
```
Comment (aanil) [inline data/report_templates/project_summary.md:216]: This [page](https://www.naiss.se/policies/acknowledge/) says `Acknowledge both NAISS and SNIC if resources were used before and after 2023-01-01.`

Comment (sofiahag) [reply data/report_templates/project_summary.md:216]: Pär will check!

Comment (aanil) [reply data/report_templates/project_summary.md:216]: @parlundin This is the other PR

**Review by parlundin — APPROVED** [review]

**Review by parlundin — APPROVED** [review]

**Review by aanil — APPROVED** [review]
👍

**Pull request merged**